### PR TITLE
Update cosmwasm_enabled for Oraichain

### PR DESCRIPTION
**Pull request by ducphamle2**

### CODE REVIEW

File: oraichain/chain.json [hunk 1]
```diff
@@ -33,38 +31,21 @@
...
-        "binaries": {
-          "linux/amd64": "https://orai.s3.us-east-2.amazonaws.com/v0.40.3/oraid"
-        },
-	"next_version_name": "v0.41.6"
-      },
-      {
-        "name": "v0.41.6",
-        "height": 13567875,
-        "proposal": 197,
+        "name": "v0.41.7",
```
Comment (JeremyParish69) [inline oraichain/chain.json:43]: Please keep the v0.40.3 version object in this array above this new v0.41.7 version object

Comment (meomeocoj) [reply oraichain/chain.json:68]: @JeremyParish69 hi, i've just updated the legacy version info. Please let me know if anything info missing. Thanks.

Comment (JeremyParish69) [reply oraichain/chain.json:68]: Thanks, but not quite right. Versions that are compatible should only take up on version object, so v0.41.3, v0.41.4, v0.41.5, and v0.41.7, are one object, as in:
```
{
  "name":"v0.41",
  "height": 12353514,
  "proposal": 185,
  "recommended_version": "v0.41.7",
  "compatible_versions": ["v0.41.3","v0.41.4","v0.41.5","v0.41.7"],
  "previous_version_name": "v0.40"
  "next_version_name": ""
},
```

What I was asking for was to not remove v0.40

Comment (ducphamle2) [reply oraichain/chain.json:68]: @meomeocoj lets re-add the older versions and link them together via previous_version_name & next_version_name

Comment (meomeocoj) [reply oraichain/chain.json]: hi @JeremyParish69, i think i've understood what you say. Please check the PR again. Thanks

**Review by JeremyParish69 — APPROVED** [review]

**Pull request merged**